actions/command.js

```diff
@@ -77,7 +77,7 @@ export function executeCommand(message, args) {
                 browserHistory.push(`${teamUrl}/channels/${redirectChannel}`);
 
                 dispatch(savePreferences(currentUserId, [{category, name, user_id: currentUserId, value: 'false'}]));
-                if (isFavoriteChannel(state, channel)) {
+                if (isFavoriteChannel(state, channel.id)) {
```

Comment:
is this related to your work? Seems unrelated?

Reply:
Not related. All of the changes that have `isFavoriteChannel` are me fixing things that someone broke on master.